src/ProjectForm.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import { Card, CardContent, Stack, TextField, Button } from '@mui/material';
+import Paper from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+import { DesktopDatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
+import moment from 'moment';
```

Comment:
`moment` is marked as a "legacy" project. You can read more about it [here](https://momentjs.com/docs/). It would soon be no longer maintained. I would consider using a different library.